src/animation_commands_extra.c

```diff
@@ -379,20 +555,13 @@ u32 sub_8004518(u16 num)
     return result;
 }
 
-#define ReadInstruction(script, cursor) ((void *)(script) + (cursor * sizeof(s32)))
-
 s32 sub_8004558(Sprite *sprite)
 {
-    if (sprite->unk21 != sprite->variant || sprite->unk1E != sprite->graphics.anim) {
```

Comment:
IMO this would be better to make this 2 macros?

```c
if (AnimUpdated(sprite)) {
  SWITCH_ANIM(sprite);
}
```

Reply:
Yeah, having an if in a macro isn't optimal in this case. You can change it if you want!